build.gradle

```diff
@@ -134,6 +134,7 @@ dependencyCheck {
 }
 
 ext {
+  lombok          = '1.18.24'
```

Comment:
Could you change the name to lombokVersion to be consistent with most of the other variables defined here?  Also remove all but one of the spaces before the equals sign.

Reply:
Confirmed variable has been changed as requested.